lib/api/uikit/functions.php

```diff
@@ -168,6 +161,56 @@ function beans_uikit_dequeue_theme( $id ) {
 	unset( $_beans_uikit_enqueued_items['themes'][ $id ] );
 }
 
+/**
+ * Gets all of the UIkit components for the given type, i.e. for core or add-ons.
```

Comment:
Get

Reply:
One of these days I'll remember that we standardized on "Get" and not "Gets." Doh!